README.md

```diff
@@ -18,3 +18,28 @@ usage: run_experiment.py [-h] [-j THREADS] experiment_yaml
 run_experiment.py: error: the following arguments are required: experiment_yaml
 ```
 
+**How to Install BFASST** 
+
+* Clone the github repository into your home directory in linux. Repository is found at https://github.com/byuccl/bfasst. Use ```git clone https://github.com/byuccl/bfasst.git.``` (Note: if Git is not installed on your device, use ```sudo apt install git``` first.) 
+
+* Confirm that the version of Vivado in _/bfasst/scripts/bfasst/config.py_ is the version installed on your computer. If not, change it to the version you have.
+
+* Use ```make capnproto_java``` 
+
+* Use ```make rapidwright```
+
+* Use ```make env```
+
+* Use ```source env.sh```
+
+* Use ```make install``` 
+
+Finally, test to confirm that everything worked correctly! Run ```source env.sh``` and ```./scripts/run_experiment.py experiments/verify_fasm_to_bels.yaml``` from the bfasst directory to check if everything works correctly.
```

Comment:
Let's just use `run_design` here instead of `run_experiment`.

Reply:
Also should this be run using the python virtual environment instead of `./`?

Reply:
Ok, I have it defaulting to run in a python virtual environment and running the check on an add4.